### ISSUE TITLE
🐛 Can't add a new work experience

### ISSUE THREAD

**Issue by maxneuvians** [GCTC-NTGC/gc-digital-talent#11749]
## 🐛 Bug

After the update last night I can no longer add Work Experience to my profile. I get an error back from the GraphQL endpoint:

```
{
	"errors": [
		{
			"message": "Variable \"$id\" of non-null type \"ID!\" must not be null.",
			"locations": [
				{
					"line": 1,
					"column": 36
				}
			]
		}
	]
}
```

Specifically I tried to add a certification experience. I can still edit existing experience however.

## 🦋 Expected Behaviour

A new work experience should be added

## 🕵️ Details

- **Operating System**: Ubuntu 24.04
- **Browser**: FireFox 131.0.2

## 📋 Steps to Reproduce

1. Add a new work experience
2. Fill in details
3. Submit

## 📸 Screenshot

![image](https://github.com/user-attachments/assets/04ee4c38-5b89-4069-8389-03821ddd1271)

**Comment by esizer** [comment]
@tristan-orourke  I think this was an issue with a null coalescing operator change we recently made. We use the same form for adding and editing, if we have an ID we expect it to be editing an experience and run that mutation.

We were passing in an empty string if we had no ID which will not fall through `??` but would for `||`. So it saw the empty string and tried passing that which graphql treats as null.

This was fixed recently in https://github.com/GCTC-NTGC/gc-digital-talent/pull/11719

We will need to test to confirm, but I bneleive this was the issue and has been fixed just not deployed yet.

https://github.com/GCTC-NTGC/gc-digital-talent/blob/62e91d475ab8d72281817bf7cb5a9f2c2b1fa64f/apps/web/src/pages/Profile/ExperienceFormPage/ExperienceFormPage.tsx#L685

https://github.com/GCTC-NTGC/gc-digital-talent/blob/62e91d475ab8d72281817bf7cb5a9f2c2b1fa64f/apps/web/src/pages/Profile/ExperienceFormPage/ExperienceFormPage.tsx#L325
